packages/did/src/DidDetails/LightDidDetails.ts

```diff
@@ -94,43 +93,4 @@ export class LightDidDetails extends DidDetails {
       this.keyRelationships.keyAgreement = [`${this.didUri}#encryption`]
     }
   }
-
```

Comment:
Not needed anymore since they are already implemented in the abstract super class `DidDetails`.